### ISSUE TITLE
Expected an error when accessing property of uninitialized class member and strictNullChecks: true

### ISSUE THREAD

**Issue by richardm** [Microsoft/TypeScript#13795]
**TypeScript Version:**  2.1.5

With strictNullChecks: true

**Code**

```ts
interface Person {
    firstName: string;
    lastName: string;
}

class Greeter {
    person: Person;

    constructor() {
      //
    }

    getFirstName () {
        return this.person.firstName;
    }
}

let greeter = new Greeter();

console.log(greeter.getFirstName());

```

**Expected behavior:**
error TS2531: Object is possibly null

**Actual behavior:**
No compile errors.
Runtime error:
Uncaught TypeError: Cannot read property 'firstName' of undefined

Person is clearly never defined, yet there are no TypeScript warnings.

**Comment by RyanCavanaugh** [comment]
Duplicate #8476
